README.md

```diff
@@ -98,4 +98,12 @@ On OpenBSD -current and 6.7+, you can use the port:
 pkg_add age
 ```
 
+On Arch Linux there are [AUR](https://aur.archlinux.org) packages for age and age-git.
+```
+auracle clone age       # or whichever AUR helper you use
```

Comment:
If you are going to suggest the full manual build method why suggest an AUR helper here at all? Just show a regular git clone. Those that want to use helpers often use ones that to all the other steps too. For example `yay -S age` ... don.